.github/workflows/deploy-test.yml

```diff
@@ -0,0 +1,67 @@
+name: Deployment to aws test environment
```

Comment:
Wondering if there's any way to dedupe this deployment code, can we extract the tag name somehow? Tracking changes between our deployment workflows will get tough as time goes on

Reply:
we have different environment setup for this pipelines...the branch deployment will cause issue ...

Reply:
we can not have conditional code in `environment.name`